README.md

```diff
@@ -28,11 +28,19 @@ git clone https://github.com/SovereignCloudStack/rookify
 cd rookify
 ```
 
-3. __TODO:__ Install script
+3. To install the local development environment
+(_Note: This will install pre-commit in your local user context_):
+```bash
+make setup
+```
+
+4. __TODO:__ Install script for container-based setup
 
 ## Usage
 __TODO__
 
+See the [Makefile](./Makefile). You can run it by simply typing `make`, which will give you a list of helpfull commands that can be used for development.
```

Comment:
I don't think that we should instruct the user to read the Makefile. I would write somthing like:
> Type `make` to get a list of available development specific commands.

Reply:
ok